spec/views/two_factor_authentication/otp_verification/show.html.slim_spec.rb

```diff
@@ -258,35 +287,31 @@
 
     context 'when users phone number is unconfirmed' do
       it 'has a link to choose a new phone number' do
-        data = presenter_data.merge(
-          unconfirmed_phone: true,
-          reenter_phone_number_path: 'some/path'
-        )
+        data = presenter_data.merge(unconfirmed_phone: true)
 
-        @presenter = TwoFactorAuthCode::PhoneDeliveryPresenter.new(data)
+        @presenter = TwoFactorAuthCode::PhoneDeliveryPresenter.new(
+          data: data,
+          view: view
+        )
 
         render
 
-        expect(rendered).to have_link(
-          t('forms.two_factor.try_again'), href: @presenter.reenter_phone_number_path
-        )
+        expect(rendered).to have_link(t('forms.two_factor.try_again'), href: verify_phone_path)
```

Comment:
Ah, I see. Before it was a method with logic to determine the right path, not an actual path.